### PR TITLE
CSTM-228: Adding list and delete commands to ui-plugins command group

### PR DESCRIPTION
**Pull request by jeremy-gooch-sp**
## Description
This PR adds the `list` and `delete` commands to the ui-plugins command group. Deletes are gated behind a verify confirmation (bypassed with --force). Delete can be by alias or by plugin id. If duplicate aliases are found the CLI errors out with a message to the end user. This happens regardless of the --force flag.

This PR also updates the plugins to use the new versioned endpoint.

## How Has This Been Tested?
There is an MV at CSTM-310 that documents the tests run including the edge cases.

### CODE REVIEW

**Review by philip-ellis-sp — APPROVED** [review]

File: cmd/ui_plugins/create.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 var createHelp string
 
 const (
-	pluginInstancesEndpoint = "/v2026/ui-plugins"
+	pluginInstancesEndpoint = "/ui-plugins/v1"
```
Comment (patrick-dowd-sp) [inline cmd/ui_plugins/create.go:24]: Out of curiosity - why are we avoiding the v2026 convention? Is it that our OSS products have a different convention?

**Review by philip-ellis-sp — APPROVED** [review]

**Review by patrick-dowd-sp — APPROVED** [review]
Looks good to me

File: cmd/ui_plugins/create.go (hunk 1 above)
Comment (jeremy-gooch-sp) [reply cmd/ui_plugins/create.go:24]: Good question. This updated path is to use the new version of the API that SP is switching to.
